packages/components/src/sandbox/index.native.js

```diff
@@ -239,6 +243,21 @@ function Sandbox( {
 		return '<!DOCTYPE html>' + renderToString( htmlDoc );
 	}
 
+	const getInjectedJavaScript = useCallback( () => {
+		// Allow parent to override the resize observers with prop.customJS (legacy support)
+		let injectedJS = customJS || observeAndResizeJS;
+
+		// Add any event listeners that were passed in.
+		Object.keys( onWindowEvents ).forEach( ( eventType ) => {
+			injectedJS += `
+				window.addEventListener( '${ eventType }', function( event ) {
+					window.ReactNativeWebView.postMessage( JSON.stringify( { type: '${ eventType }', ...event.data } ) );
```

Comment:
I was going for the more standard DOM event terminology since `type` is a property of the event. Also the use of `action` is internal to the Sandbox component so I don't see it as being inconsistent to the owners of a Sandbox instance. I'm fine changing it but when registering event listeners in the props I would expect to see `event.type` coming from a DOM event.

Reply:
Ah, good point. In that case, let's use the DOM event terminology 👍.